# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/test/webgl-conf/mochitest-errata.ini
# Commit: 7e2a2b1b416f
# Full Hash: 7e2a2b1b416f1a64c6548aab29e92acc9554ae7d
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Fix all webgl regression tests according to CI. r=handyman
#   
#   (This is a combination of 31 commits)
#   
#   * Fix Linux compilation.
# ==============================================================================

diff -r ccfa767dba64 -r 7e2a2b1b416f dom/canvas/test/webgl-conf/mochitest-errata.ini
--- a/dom/canvas/test/webgl-conf/mochitest-errata.ini	Wed Jan 08 22:19:16 2020 +0000
+++ b/dom/canvas/test/webgl-conf/mochitest-errata.ini	Wed Jan 08 22:19:23 2020 +0000
@@ -76,9 +76,6 @@
 
 ########################################################################
 # Global
-[generated/test_2_conformance__extensions__webgl-compressed-texture-s3tc.html]
-# getError expected: NO_ERROR. Was INVALID_OPERATION : uploading a texture from a PBO subrange
-fail-if = (os != 'android')
 [generated/test_2_conformance2__extensions__ovr_multiview2.html]
 fail-if = (os == 'win')
 # Test fixed upstream, just needs revendor.
@@ -829,7 +826,6 @@
 # [task 2019-12-27T02:30:56.787Z] 02:30:56     INFO - GECKO(1114) | #04: mozilla::WebGLContext::PresentScreenBuffer(mozilla::gl::GLScreenBuffer*) [dom/canvas/WebGLContext.cpp:1430]
 # [task 2019-12-27T02:30:56.788Z] 02:30:56     INFO - GECKO(1114) | #05: mozilla::WebGLContextUserData::PreTransactionCallback(void*) [dom/canvas/WebGLContext.cpp:1154]
 
-
 ########################################################################
 ########################################################################
 # Mac
@@ -1138,9 +1134,6 @@
 skip-if = (os == 'win')
 [generated/test_2_conformance2__query__occlusion-query.html]
 skip-if = (os == 'win')
-[generated/test_2_conformance2__query__query.html]
-# random timed out
-skip-if = (os == 'win')
 [generated/test_2_conformance2__attribs__gl-vertex-attrib.html]
 # random timed out
 skip-if = (os == 'win')